safe/gui/widgets/dock.py

```diff
@@ -449,12 +449,15 @@ def getting_started_message(self):
         basics_list.add(m.Paragraph(
             self.tr(
                 'Make sure you have defined keywords for your hazard and '
-                'exposure layers. You can do this using the keywords icon '),
+                'exposure layers. You can do this using the keywords '
+                'editor icon '),
+            m.Image('file:///%s/img/icons/show-keyword-editor.svg' %
+                    (resources_path()), **SMALL_ICON_STYLE),
+            self.tr(' or keywords creation wizard '),
             m.Image(
-                'file:///%s/img/icons/show-keyword-editor.svg' % (
-                    resources_path()),
-                **SMALL_ICON_STYLE),
-            self.tr(' in the InaSAFE toolbar.')))
+                'file:///%s/img/icons/show-keyword-wizard.svg' %
```

Comment:
Hi @lucernae 

Is it not reversed? Keyword Wizard is in toolbar, while Keyword Editor is in menu bar.